codecarbon/emissions_tracker.py

```diff
@@ -303,10 +303,7 @@ def __init__(
             hardware = CPU.from_utils(self._output_dir, "intel_rapl")
             self._hardware.append(hardware)
             self._conf["cpu_model"] = hardware.get_model()
-        else:
```

Comment:
I arrived to this case. (Testing with Windows machine, but unfortunately I do not have RAPL in this machine and the Intel Power Gadget is not even installed, see #457).
Why remove this else?

Reply:
Just for testing both mode.